decidim-core/config/locales/en.yml

```
@@ -1111,6 +1111,8 @@ en:
         translated_text: 'Automatically translated text:'
     notifications:
       no_notifications: No notifications yet.
+      show:
+        missing_event: 'Missing event: This resource is no longer available.'
```

This is too technical and developer-speak. How about changing it to something a bit more friendly? 

```suggestion
        missing_event: Oops, this notification belongs to an item that is no longer available. You can discard it.
```

What do you think @microstudi?

---

love it!